R/get_plot_component.R

```diff
@@ -0,0 +1,70 @@
+#' Get plot components
+#'
+#' Extract plot components from a ggplot or gtable. `get_plot_component()`
+#' extracts grobs or a list of grobs. `plot_component_names()` provides the
+#' names of the components in the plot. `plot_components()` returns all
+#' components as a list.
+#'
+#' @param plot A ggplot or gtable to extract from.
+#' @param pattern The name of the component.
```

Comment:
Could you add one or two examples here explaining what a name of a component might be?